client/src/App.vue

```diff
@@ -1092,100 +1093,48 @@
-									<i class="fas fa-user-check"></i>).
-								</li>
-								<li>
-									Once all confirmed, the session owner launches the desired game mode.
-								</li>
-							</ol>
-						</div>
-					</div>
 					<div class="container">
 						<div class="section-title">
-							<h2>Collection Import</h2>
+							<h2>News</h2>
 						</div>
 						<div class="welcome-section">
-							Each player can import their MTGA collection to restrict the card pool to cards they own.
-							(Session owners can bypass this feature by enabling "Ignore Collections"):
-							<ol>
-								<li>
-									Enable "Detailed Logs" in MTG Arena. It is required for the collection import to
-									work. The toggle can be found in "Options > Account > Detailed Logs (Plugin
-									Support)".
-								</li>
-								<li>
-									<a onclick="document.querySelector('#file-input').click()">Upload</a>
-									your MTGA log file "Player.log" located in
-									<tt
-										class="clickable"
-										@click="logPathToClipboard"
-										v-tooltip="'Copy path to clipboard'"
-										>C:\Users\%username%\AppData\LocalLow\Wizards Of The Coast\MTGA\</tt
-									>
-									(Note:
-									<a
-										href="https://support.microsoft.com/en-us/help/14201/windows-show-hidden-files"
-										target="_blank" rel="noopener nofollow"
-									>
-										AppData folder is hidden by default
-										<i class="fas fa-external-link-alt"></i>
-									</a>
-									).
-								</li>
-							</ol>
+							<div class="news">
+								<em>April 04, 2021</em>
+								<p><img src="img/sets/stx.svg" class="set-icon" style="--invertedness: 100%" /> Strixhaven: School of Mages support!</p>
+							</div>
+							<div class="news">
+								<em>March 28, 2021</em>
+								<p>
+									Introducing the <a href="https://www.mtgadraft.tk">MTGADraft.tk</a> domain.<br />
+									<i class="fas fa-exclamation-triangle yellow"></i> This is the exact same website but logs and preferences do not transfert over domains, you can still retrieve your game logs by accessing the <a href="https://mtgadraft.herokuapp.com">old URL</a>.  
+								</p>
+							</div>
+							<div class="news">
+								<em>March 06, 2021</em>
+								<p>
+									<img src="img/sets/tsr.svg" class="set-icon" style="--invertedness: 100%" />
+									Time Spiral Remastered (TSR) is now available! (see the "<i class="fas fa-ellipsis-h"></i> More sets..." option)<br />
+								</p>
+							</div>
 						</div>
 					</div>
+				</div>
+				<div class="welcome-col">
 					<div class="container">
 						<div class="section-title">
 							<h2>Help</h2>
 						</div>
 						<div class="welcome-section">
-							Visit the
-							<a @click="displayedModal = 'help'">FAQ / Help</a>
-							section.
-							<br />For any question/bug report/feature request you can email to
+							<a @click="displayedModal = 'gettingStarted'"><i class="fas fa-question-circle"></i> Get Started</a> guide.<br />
```

Comment:
I think the `(i)` is more suitable here: https://fontawesome.com/icons/info-circle?style=solid

```suggestion
							<a @click="displayedModal = 'gettingStarted'"><i class="fas fa-info-circle"></i> Get Started</a> guide.<br />
```